webpack.config.js

```diff
@@ -5,6 +5,11 @@ var definePlugin = new webpack.DefinePlugin({
     __DEBUG__: JSON.stringify(process.env.NODE_ENV === 'development'),
 });
 
+var providePlugin = new webpack.ProvidePlugin({
```

Comment:
With this reasoning, maybe fetch could follow the same pattern, as it's not directly exposed. But let's not do that here anyway.